include/circt/Dialect/OM/OMPasses.td

```diff
@@ -25,3 +25,11 @@ def LinkModules: Pass<"om-link-modules", "mlir::ModuleOp"> {
   }];
   let constructor = "circt::om::createOMLinkModulesPass()";
 }
+
+def VerifyObjectFields: Pass<"om-verify-object-fields"> {
```

Comment:
The pass is intentionally unrestricted so that we can schedule the pass on build.module and firrtl.circuit.

Reply:
Instead of the assert, could use `canScheduleOn` to only run on what's supported? Just a thought.

Reply:
Oh nice, I didn't know about `canScheduleOn`.

Reply:
Yeah, thanks. It looks working well!